megamek/src/megamek/common/MechView.java

```diff
@@ -453,19 +459,19 @@ public MechView(Entity entity, boolean showDetail, boolean useAlternateCost, boo
             sFluff.add(list);
         }
         
-        if (entity.getFluff().getOverview() != "") {
+        if (!entity.getFluff().getOverview().equals("")) {
             sFluff.add(new LabeledElement("Overview", entity.getFluff().getOverview()));
         }
         
-        if (entity.getFluff().getCapabilities() != "") {
+        if (!entity.getFluff().getCapabilities().equals("")) {
             sFluff.add(new LabeledElement("Capabilities", entity.getFluff().getCapabilities()));
         }
         
-        if (entity.getFluff().getDeployment() != "") {
+        if (!entity.getFluff().getDeployment().equals("")) {
             sFluff.add(new LabeledElement("Deployment", entity.getFluff().getDeployment()));
         }
         
-        if (entity.getFluff().getHistory() != "") {
+        if (!entity.getFluff().getHistory().equals("")) {
```

Comment:
This is better, but I would suggest String#isEmpty

Reply:
I agree, changed them all to isEmpty calls instead.